spec/models/user_spec.rb

```diff
@@ -125,9 +127,23 @@
       last_name: 'Doe',
       username: 'jdoe',
       phone_number: '0449 774 759',
-      email: 'Foo@ExAMPle.CoM'
+      email: 'Foo@ExAMPle.CoM',
+      password: 'foobar',
+      password_confirmation: 'foobar'
     )
 
     expect(user.email).to eq(test_email.downcase)
   end
+
+  # BCrypt Authentication
+
+  it 'password should be present (non-blank)' do
+    subject.password = subject.password_confirmation = " " * 6
```

Comment:
are we checking password/confirmation against blank? Is it wrong to have six spaces as a password?

Reply:
This was my interpretation of the tutorial example into Rspec, from my understanding that is indeed what they were attempting to verify / test.